tests/integration/src/test/scala/mesosphere/marathon/integration/TaskUnreachableIntegrationTest.scala

```diff
@@ -263,7 +264,9 @@ class TaskUnreachableIntegrationTest extends AkkaIntegrationTest with EmbeddedMa
 
       And("a new pod with is scheduled")
       waitForStatusUpdates("TASK_RUNNING")
-      marathon.status(pod.id).value.instances should have size 1
+      eventually {
+        marathon.status(pod.id).value.instances should have size 1
```

Comment:
How is this a fix? Is `instances.size == 2` here? And if it is - why? Wasn't the other instance supposed to be `wipe`d above?

Reply:
What is the error message on a failure?

Reply:
`
List(PodInstanceStatus(simple-pod-with-one-instance-wipe-test.instance-97065cb2-af6e-11e8-98b6-0242d605685d,STABLE,2018-09-03T11:43:35.310Z,None,List(),Some(172.16.10.245),Some(d66d7e98-bcdf-4d8c-951f-0a2af8f827a1-S1),None,None,Some(Resources(0.2,64.0,10.0,0)),List(NetworkStatus(None,List(172.16.10.245))),List(ContainerStatus(task1,TASK_RUNNING,2018-09-03T11:43:35.310Z,None,List(),Some(simple-pod-with-one-instance-wipe-test.instance-97065cb2-af6e-11e8-98b6-0242d605685d.task1),List(),Some(Resources(0.1,32.0,0.0,0)),None,2018-09-03T11:43:35.310Z,2018-09-03T11:43:35.310Z)),Some(/v2/pods/simple-pod-with-one-instance-wipe-test::versions/2018-09-03T11:43:30.198Z),List(),2018-09-03T11:43:35.310Z,2018-09-03T11:43:35.310Z), PodInstanceStatus(simple-pod-with-one-instance-wipe-test.instance-97179ac3-af6e-11e8-98b6-0242d605685d,PENDING,2018-09-03T11:43:35.099Z,None,List(),Some(172.16.10.245),Some(d66d7e98-bcdf-4d8c-951f-0a2af8f827a1-S1),None,None,Some(Resources(0.1,32.0,10.0,0)),List(),List(ContainerStatus(task1,TASK_STAGING,2018-09-03T11:43:35.099Z,None,List(),Some(simple-pod-with-one-instance-wipe-test.instance-97179ac3-af6e-11e8-98b6-0242d605685d.task1),List(),None,None,2018-09-03T11:43:35.099Z,2018-09-03T11:43:35.099Z)),Some(/v2/pods/simple-pod-with-one-instance-wipe-test::versions/2018-09-03T11:43:30.198Z),List(),2018-09-03T11:43:35.099Z,2018-09-03T11:43:35.099Z)) had size 2 instead of expected size 1
`